internal/pkg/fieldpropagator/analyzer.go

```diff
@@ -119,10 +119,8 @@ func fieldAddr(x ssa.Value) (*ssa.FieldAddr, bool) {
 	return nil, false
 }
 
-type FieldPropagators map[types.Object]bool
-
 // Contains determines whether a call is a field propagator, i.e. whether it is contained by the FieldPropagators map.
-func (f FieldPropagators) Contains(c *ssa.Call) bool {
+func (r ResultType) Contains(c *ssa.Call) bool {
```

Comment:
Nit: consider keeping the method-set near the type declaration.

Reply:
Good point! Done.